wsmaster/codenvy-hosted-api-admin/src/main/java/com/codenvy/api/user/server/UserServicePermissionsFilter.java

```diff
@@ -66,6 +67,10 @@ protected void filter(GenericResourceMethod genericResourceMethod, Object[] argu
                 //public methods
                 return;
             case "create":
+                if (!licenseManager.hasAcceptedFairSourceLicense()) {
```

Comment:
It looks redundant for me because LicenseFilter should deny request when license is not accepted. No?